grease-cli/tests/sanity/pass/test-pointer-offset/test.c

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) Galois, Inc. 2024 */
+
+// x64: flags {"--symbol", "test", "--no-heuristics", "--initial-precondition", "tests/sanity/pass/test-pointer-offset/init.shape"}
+// x64: go(prog)
+
+struct foo {
+    char* begin;
+    char* end;
+};
+
+
+
```

Comment:
Very minor: Any particular reason to have so many newlines? Makes the file bigger without a substantial gain in readability IMO.
```suggestion

```